espnet2/bin/asr_inference.py

```diff
@@ -128,8 +129,8 @@ def __init__(
                     "torch version < 1.5.0. Switch to qint8 dtype instead."
                 )
 
-        quantize_modules = set([getattr(torch.nn, q) for q in quantize_modules])
-        quantize_dtype = getattr(torch, quantize_dtype)
+        qconfig_spec = set([getattr(torch.nn, q) for q in quantize_modules])
+        quantize_dtype: torch.dtype = getattr(torch, quantize_dtype)
```

Comment:
Is it OK?
I just want to confirm.

Reply:
Which line 132? 133? In 132, I changed the variable to avoid `TypeCheckError`, and also renamed subsequently.
In case of L133, I needed to change the type of variable (explicitly) to avoid the same error. 
Both cases are allowed and the tests did not raise any error after these modifications